cg/smart_contracts/ClearGDPR.sol

```diff
@@ -1,17 +1,26 @@
-pragma solidity 0.4.21; 
+/*  ClearGDPR smart contract version 6
```

Comment:
Now we only have one version of the smart contract, with the most important remarks and comments